COMPILING.md

```diff
@@ -87,9 +87,17 @@ git clone --recursive https://github.com/cuberite/cuberite.git
 
 ### Building ###
 
-Follow the instructions at [CMake on Unix-based platforms](#cmake-on-unix-based-platforms).
+Run the following commands to build Cuberite:
+
+```
+mkdir Release
+cd Release
+cmake -DCMAKE_BUILD_TYPE=RELEASE ..
+make -j`nproc`
```

Comment:
I don't think so, because it's still limited by the number of jobs that can actually run together; only jobs that don't depend on each other can be run in parallel. In Cuberite, this mostly means the external libraries, so it's like 10 jobs max, give or take. Should be okay.